apps/researcher/src/app/[locale]/objects/[id]/object-lists-menu.tsx

```diff
@@ -0,0 +1,147 @@
+'use client';
+
+import {useTranslations} from 'next-intl';
+import {Fragment, useState, useEffect} from 'react';
+import {Menu, Transition} from '@headlessui/react';
+import {ChevronDownIcon} from '@heroicons/react/20/solid';
+import {useUser} from '@clerk/nextjs';
+import {getCommunityLists, addObjectToList} from './database-actions';
+import {SelectObjectList} from '@colonial-collections/database';
+import {useNotifications} from 'ui';
+
+interface CommunityMenuItemsProps {
+  communityId: string;
+  objectId: string;
+  userId: string;
+}
+
+function CommunityMenuItems({
+  communityId,
+  objectId,
+  userId,
+}: CommunityMenuItemsProps) {
+  const [objectLists, setObjectLists] = useState<SelectObjectList[]>([]);
+  const t = useTranslations('ObjectDetails');
+  const {addNotification} = useNotifications();
+
+  async function addObjectToListClick(objectList: SelectObjectList) {
+    try {
+      await addObjectToList({
+        listItem: {
+          objectIri: objectId,
+          objectListId: objectList.id,
+          createdBy: userId,
+        },
+        communityId,
+      });
+
+      addNotification({
+        id: 'objectAddedToList',
+        message: t.rich('objectAddedToList', {
+          name: () => <em>{objectList.name}</em>,
+        }),
+        type: 'success',
+      });
+    } catch (err) {
+      if ((err as Error).message.includes('Duplicate entry')) {
```

Comment:
This does not work in production, I will fix this on Monday. The response in production only: 

Error: An error occurred in the Server Components render. The specific message is omitted in production builds to avoid leaking sensitive details. A digest property is included on this error instance which may provide additional details about the nature of the error.